recipes-multimedia/gstreamer/gstreamer1.0-plugins-bad_%.bbappend

```diff
@@ -1,2 +1,2 @@
-PACKAGECONFIG_remove = "${@bb.utils.contains('DISTRO_FEATURES', 'wayland', 'wayland', '', d)}"
```

Comment:
I think the logic should be reversed instead of complete removal
`PACKAGECONFIG_remove = "${@bb.utils.contains('DISTRO_FEATURES', 'wayland', '', 'wayland', d)}"`

Reply:
@kraj Sorry for the delay, I forgot about this pull request. I've updated like you suggested and re-based onto master